src/relay/backend/interpreter.cc

```diff
@@ -448,7 +450,7 @@ class Interpreter :
                     "fusing and lowering";
     }
     if (auto con = call->op.as<ConstructorNode>()) {
-      return ConstructorValueNode::make(GetRef<Constructor>(con), args);
+      return ConstructorValueNode::make(con->tag, args);
```

Comment:
can you pass in the optional constructor to help debug?

Reply:
done